rspack.iframe-sdk-embed.config.js

```diff
@@ -19,6 +19,8 @@ const SDK_SRC_PATH = __dirname + "/enterprise/frontend/src/embedding-sdk";
 const OUT_FILE_NAME = "embed.js";
 const OUT_TEMP_PATH = path.resolve(BUILD_PATH, "tmp-embed-js");
 
+const DEV_PORT = process.env.PORT || 8080;
```

Comment:
Nice, now this points to the dev-server hosted on the main `rspack.config.js` as they share the same dev server I assume 👍🏻